src/toil_scripts/batch_alignment/README.md

```diff
@@ -2,8 +2,9 @@
 ### Guide: Running the BWA Pipeline using Toil
 
 This guide attempts to walk the user through running this pipeline from start to finish. If there are any questions
-please contact John Vivian (jtvivian@gmail.com). If you find any errors or corrections please feel free to make a 
-pull request.  Feedback of any kind is appreciated.
+please contact the Toil team at: Toil@googlegroups.com 
```

Comment:
Move to project level readme, no? We should have it there, and not duplicate it across all pipeline readmes.